src/main/java/domain/Lotto.java

```diff
@@ -40,11 +41,17 @@ public List<Integer> getLottoNumbers() {
                 .collect(Collectors.toList());
     }
 
-    public int calculateMatchCount(Lotto winningLotto) {
+    public int calculateMatchCount(WinningLotto winningLotto) {
         return (int) lottoNumbers.stream()
                 .map(LottoNumber::getLottoNumber)
-                .filter(winningLotto.getLottoNumbers()::contains)
+                .filter(lottoNum -> winningLotto.getLottoNumbers().contains(lottoNum) || lottoNum == winningLotto.getBonusNumber())
                 .count();
     }
 
+    public boolean isMatchBonusNumber(WinningLotto winningLotto) {
+        return lottoNumbers.stream()
+                .map(LottoNumber::getLottoNumber)
+                .anyMatch(lottoNumber -> lottoNumber.equals(winningLotto.getBonusNumber()));
+    }
```

Comment:
이름을 그냥 contains로 만들면 단순히 보너스 번호뿐 아니라 좀 더 범용적으로 사용가능할 것 같아요 ㅎㅎ.
아니면 네이밍 시그니처를 좀 더 돋보이게 한다고해도 이렇게 위임할 수 있을 것 같네요. 

```suggestion
    public boolean isMatchBonusNumber(WinningLotto winningLotto) {
        return contains(winningLotto.getBonusNumber());
    }
    
    public boolean contains(LottoNumber number) {
        return lottoNumbers.contains(number);
    }
```

Reply:
와.. 아직 스트림함수를 야무지게(?) 사용하는게 너무 힘든 것 같네요ㅠㅠ
이렇게 깔끔한 형태로 가능하다니...ㅠㅠ!!!